packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -670,12 +670,12 @@ export class DeltaManager
                     }
 
                     const retryDelayFromError = getRetryDelayFromError(origError);
-                    delay = retryDelayFromError ?? Math.min(delay * 2, MaxReconnectDelaySeconds);
```

Comment:
retryDelayFromError  is in seconds, but delay & MaxReconnectDelayInMs are in ms.
You can't mix them (without multiplying here or in getRetryDelayFromError) by 1000.

Reply:
From what I can see, getRetryDelayFromError is used only in loader & drivers. I think you can safely change it to return time in ms.

BTW, this code in getFileLink.ts is wrong (has same problem):

                retryAfter = getRetryDelayFromError(err) ?? Math.min(retryAfter * 2, 8000);
                await delay(retryAfter);